test/index.js

```diff
@@ -85,12 +85,26 @@ describe('AppsFlyer', function() {
       test.invalid(msg, settings);
     });
 
-    it('should be invalid if you do not send an advertisingId', function() {
+    /**
+    * Temporarily skipping this test per: https://segment.zendesk.com/agent/tickets/73014.
+    * Despite what their documentation says, AppsFlyer's API will accept events wihtout an
+    * advertisingId / idfa property. This ensure block was causing issues for customers
+    * that have been using this integration without sending advertisingId previously.
+    * AppsFlyer will be working to migrate people away from doing this in the next couple of months.
+    *
+    * @ccnixon 05/10/2017
+    */
+
+    it.skip('should be invalid if you do not send an advertisingId', function() {
```

Comment:
this test should be modified so that it should be invalid if you send mobile && !advertisingId but valid if server side data && !advertisingId instead of skipping the test